paths/teams/add_project.yaml

```diff
@@ -0,0 +1,57 @@
+---
+summary: Add Project
+description: Adds an existing project to the team.
+operationId: teams/projects/create
+tags:
+- Teams
+parameters:
+- "$ref": "../../parameters.yaml#/X-PhraseApp-OTP"
+- "$ref": "../../parameters.yaml#/account_id"
+- "$ref": "../../parameters.yaml#/team_id"
+responses:
+  '201':
+    description: Created
+    headers:
+      X-Rate-Limit-Limit:
+        "$ref": "../../headers.yaml#/X-Rate-Limit-Limit"
+      X-Rate-Limit-Remaining:
+        "$ref": "../../headers.yaml#/X-Rate-Limit-Remaining"
+      X-Rate-Limit-Reset:
+        "$ref": "../../headers.yaml#/X-Rate-Limit-Reset"
+  '400':
+    "$ref": "../../responses.yaml#/400"
+  '404':
+    "$ref": "../../responses.yaml#/404"
+  '429':
+    "$ref": "../../responses.yaml#/429"
+x-code-samples:
+- lang: Curl
+  source: |-
+    curl "https://api.phrase.com/v2/accounts/:account_id/teams/:team_id/projects" \
+      -u USERNAME_OR_ACCESS_TOKEN \
+      -X POST \
+      -d '{"id":"a4b3c2d1"}' \
+      -H 'Content-Type: application/json'
+- lang: CLI v2
+  source: |-
+    phrase teams projects \
```

Comment:
I don't think this is correct. It should be `phrase team projects_create` I think. Did you run the CLI generator?

Reply:
no, i haven't yet. Also i dont know how it works. I will look into that

Reply:
@rlisowski I took a look at the paths/spaces/add_project.yaml file which describes the same functionality and there it is done without the underscore. I think it is either correct like this or we should change it there as well

Reply:
Spaces docs are wrong, I will fix them in separate PR. Thanks for pointing it out.

```
$ phrase spaces -h
Spaces API

Usage:
  phrase spaces [command]

Available Commands:
  create          Create a Space
  delete          Delete Space
  list            List Spaces
  projects_create Add Project
  projects_delete Remove Project
  projects_list   List Projects
  show            Get Space
  update          Update Space
```

Reply:
@mobuffon I fixed spaces docs

Reply:
okay, thanks